magic-links/functions/start-verify.js

```diff
@@ -43,6 +43,7 @@ exports.handler = function(context, event, callback) {
   const client = context.getTwilioClient();
   const service = context.VERIFY_SERVICE_SID;
   const to = event.to;
+  const callback_url = `${context.DOMAIN}/${context.CALLBACK_PATH}`
```

Comment:
If you end up changing it to `context.DOMAIN_NAME` you'll have to add the protocol like `https://` to the string. Also in case this is being used in a nested template you'd have to consider `context.PATH`. So basically:

```suggestion
  const callback_url = `https://${context.DOMAIN_NAME}${context.PATH.substr(0, context.PATH.lastIndexOf('/'))}/${context.CALLBACK_PATH}`
```

Reply:
any suggestions for how to gracefully swap between http for localhost and https for production? I could definitely do something like:

```
const protocol = (context.DOMAIN_NAME.startsWith('localhost:') ? 'http' : 'https')
const callback_url = `${protocol}://${context.DOMAIN_NAME}${context.PATH.substr(0, context.PATH.lastIndexOf('/'))}/${context.CALLBACK_PATH}`
```

But that seems a touch clunky. Not sure I want to dedicate _environments_ for one variable either. Curious if you have suggestions.